src/Dockerfile

```
@@ -18,31 +26,28 @@ RUN apk add --no-cache \
     jq \
     coreutils \
     procps \
-    ncurses
+    ncurses \
+    binutils
 
-ARG WEB_BRANCH="development-v6"
-ARG CORE_BRANCH="development-v6"
-ARG FTL_BRANCH="development-v6"
+ADD https://ftl.pi-hole.net/macvendor.db /macvendor.db
+COPY crontab.txt /crontab.txt
 
 # download a the main repos from github
-RUN git clone --depth 1 --single-branch --branch ${WEB_BRANCH} https://github.com/pi-hole/AdminLTE.git /var/www/html/admin && \
-    git clone --depth 1 --single-branch --branch ${CORE_BRANCH} https://github.com/pi-hole/pi-hole.git /etc/.pihole
-
-# Download the latest version of pihole-FTL for alpine:
-# Probably need this to be built for different FTLARCHs
-RUN if   [ "$TARGETPLATFORM" = "linux/amd64" ];    then FTLARCH=amd64; \
+RUN git clone --depth 1 --single-branch --branch development-v6 https://github.com/pi-hole/AdminLTE.git /var/www/html/admin && \
+    git clone --depth 1 --single-branch --branch development-v6 https://github.com/pi-hole/pi-hole.git /etc/.pihole ;\
```

Why did you hard-code the branches here?

```suggestion
RUN git clone --depth 1 --single-branch --branch ${WEB_BRANCH} https://github.com/pi-hole/AdminLTE.git /var/www/html/admin && \
    git clone --depth 1 --single-branch --branch ${CORE_BRANCH} https://github.com/pi-hole/pi-hole.git /etc/.pihole ;\
```

---

my mistake, sorry.